tests/php/test-class-amp-options-manager.php

```diff
@@ -9,13 +9,14 @@
 use AmpProject\AmpWP\Tests\Helpers\AssertContainsCompatibility;
 use AmpProject\AmpWP\Tests\Helpers\LoadsCoreThemes;
 use AmpProject\AmpWP\Admin\ReaderThemes;
+use AmpProject\AmpWP\Tests\DependencyInjectedTestCase;
 
 /**
  * Tests for AMP_Options_Manager.
  *
  * @covers AMP_Options_Manager
  */
-class Test_AMP_Options_Manager extends WP_UnitTestCase {
+class Test_AMP_Options_Manager extends DependencyInjectedTestCase {
```

Comment:
`DependencyInjectedTestCase` doesn't seem to be required.

```suggestion
class Test_AMP_Options_Manager extends WP_UnitTestCase {
```